src/components/AppNavigation/CalendarList/CalendarListItem.vue

```diff
@@ -342,6 +335,16 @@ export default {
 
 			return ''
 		},
+	  /**
+	   * compute aria-description for AppNavigationItem link
+	   *
+	   * @return {string}
+	   */
+	  descriptionAppNavigationItem() {
+			return this.calendar.enabled
+				? t('calendar', 'Disable calendar "{calendar}"', { calendar: this.calendar.displayName })
```

Comment:
`this.calendar.displayName` can be undefined, see how `calendar.displayName || $t('calendar', 'Untitled calendar')` is used above.